frontend/test/__support__/ui.tsx

```diff
@@ -24,6 +24,7 @@ export interface RenderWithProvidersOptions {
   storeInitialState?: Partial<State>;
   withSampleDatabase?: boolean;
   withRouter?: boolean;
+  initialPath?: string;
```

Comment:
`initialRouterPath`? Simply `initialPath` isn't obvious. Also cc @kulyk - could you take a look and double check that this change makes sense?

Reply:
`initialRouterPath` sounds clearer. I'm with that 👍